containers/hugging-face-inference/terraform/terraform.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e
+
+# Common environment variables
+export TF_VAR_access_key=${SCW_ACCESS_KEY} \
+       TF_VAR_secret_key=${SCW_SECRET_KEY} \
+       TF_VAR_project_id=${SCW_PROJECT_ID}
+
+# Associative list of models to deploy using json data
+declare -A hf_models
+eval "$(jq -r '.[]|.[]|"hf_models[\(.file)]=\(.source)"' hf-models.json)"
+
+# Login to docker Scaleway's registry
+docker login "rg.$REGION.scw.cloud" -u nologin --password-stdin <<< "$SCW_SECRET_KEY"
+
+# Initialize, plan, and deploy each model in a Terraform workspace
+apply() {
+       terraform init
+       for model_file_name in "${!hf_models[@]}";
```

Comment:
**question(terraform):** Instead of using bash for-loops here, can we use `for_each` in the Terraform files to iterate over the list of models in a variable?